content/docs/ops/configuration-management.md

```diff
@@ -29,21 +29,21 @@ Security tests need to be executed in the staging environment where changes are
 
 ## Change workflow
 
-1. All configuration changes must flow through a git repository, centrally managed through GitHub, unless they contain sensitive information. In these cases, sensitive information should be stored in an S3 bucket with a proper security policy, encryption, and versioned, so that changes can be easily rolled back.
+1. All configuration changes must flow through a git repository, centrally managed through GitHub, unless they contain sensitive information. In these cases, sensitive information should be stored in an S3 bucket with a proper security policy and encryption, versioned such that changes can be easily rolled back.
```

Comment:
Part of the security policy determines the encryption parameters. Buckets themselves aren't encrypted, but each item in the bucket can be. You can force encryption by defining in your policy that everything uploaded to the bucket must be encrypted: https://github.com/18F/cg-provision/blob/master/terraform/modules/s3_bucket/encrypted_bucket/encrypted_bucket.tf#L20